docs/android/tips/compose_101.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Jetpack Compose 101"
+sidebar_label: "Jetpack Compose 101"
+---
+
+🚧🚧🚧 Under Construction 🚧🚧🚧
+
+## How to create a new screen in the existing app and iterate quickly
+
+To create a new screen in the app and iterate quickly, follow these steps:
+
+1. **Extract the Compose UI screen**:  
+   Create a dedicated Kotlin file for your Compose UI screen. Use the `@Preview` annotation to enable preview capabilities within the IDE. This also makes the screen compatible with [screenshot testing](../testing/screenshot_testing).
+
+2. **Leverage hot reload**:  
+   After the first build of the app, navigate to your screen. Jetpack Compose provides hot reload capabilities out of the box, allowing you to see changes in real-time. However, note that there are some limitations, such as not being able to reload changes to certain structural elements.
+
+3. **Use previews effectively**:  
+   Use multiple `@Preview` annotations to test your screen in different configurations (e.g., light/dark mode, different screen sizes). This helps ensure your UI adapts well to various scenarios.
+
+## Theme/Design system
```

Comment:
This section is specific to HA, but all the others feel like generic Compose tips to me. That may be useful but I'm not sure it has a place in the HA docs, as you need to maintain it and Compose moves fast. Why not link to Google's docs?

Reply:
which Google docs?

Reply:
these docs? https://developer.android.com/develop/ui/compose/documentation

a bit in depth but that is indeed where we started lol